vue/components/molecules/restrictions-alert/restrictions-alert.vue

```diff
@@ -10,11 +10,27 @@
                         {{ "ripe_commons.restrictions_alert.close_button" | locale }}
                     </a>
                 </div>
-                <div class="message-container">
-                    <span>
-                        {{ "ripe_commons.restrictions_alert.limited" | locale }}
-                        {{ "ripe_commons.restrictions_alert.back" | locale }}
+                <div
+                    class="message-container"
+                    v-bind:style="{ width: messages.length > 0 ? '100%' : null }"
+                >
+                    <span v-if="hasRestrictions">
+                        {{ restrictionMessage }}
                     </span>
+                    <div v-if="messages.length > 0">
+                        <div
+                            class="messages"
+                            v-for="(message, index) in messages"
+                            v-bind:key="index"

```

Comment:
Avoid using index as key when possible, since it defeats the purpose of having a key (not re-rendering everything every time). You can use name + value.